CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 4.4.2
+- Upgraded Dependencies to address security concerns (https://github.com/fauna/faunadb-js/pull/600)
```

Comment:
```suggestion
- Upgrade dependencies to address security concerns: [#600](https://github.com/fauna/faunadb-js/pull/600)
```

It would be good to apply the link suggestion to all of the other PR links.